### PR TITLE
Remove the context instance argument in views.py

### PR DESCRIPTION
**Pull request by ankitjavalkar**
- Remove the context instance argument in views.py (deprecated since Django 1.10 https://docs.djangoproject.com/en/1.10/releases/1.10/)
- Replace ```render_to_response``` with ```render```

### CODE REVIEW

File: yaksh/decorators.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from django.shortcuts import render_to_response, redirect
+from django.shortcuts import render, render_to_response, redirect
```
Comment (prathamesh920) [inline yaksh/decorators.py]: ```render_to_response``` is replaced by ```render```, then ```render_to_response``` can be removed from the import as it is not used any more.

File: yaksh/views.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 from django.http import HttpResponse, JsonResponse
 from django.core.urlresolvers import reverse
 from django.contrib.auth import login, logout, authenticate
-from django.shortcuts import render_to_response, get_object_or_404, redirect
+from django.shortcuts import render_to_response, render, get_object_or_404, redirect
```
Comment (prathamesh920) [inline yaksh/views.py]: Again ```render_to_response``` needs to be removed.

**Comment by ankitjavalkar** [comment]
Fixed changes as per suggestions

**Comment by ankitjavalkar** [comment]
I have bumped the django version from 1.9.5 to 1.10. The Context instance is deprecated since v1.10, keeps giving warnings on test-server/local servers.

**Pull request merged**